src/oop/programs_with_multiple_classes/hard/q5/Book.java

```diff
@@ -0,0 +1,18 @@
+package oop.programs_with_multiple_classes.hard.q5;
+
+import java.time.Year;
+
+public class Book {
+    private int year;
+    private String type;
+
+    public Book(int year, String type) {
+        this.year = year;
+        this.type = type;
+    }
+
+    public int determineTime() {
+        return Year.now().getValue() - year;
```

Comment:
Please add a hint for this

Reply:
Students don't need to use this. They can just put "2023." I am only doing this so the test will stay current.